data/coalition/coalition missions.txt

```diff
@@ -2335,3 +2335,469 @@ mission "Coalition: Submarines 4"
+			choice
+				`	"I take it 'inspector' is what the people sent out are called?"`
+				`	"Good to hear they had a change of heart. So, where are we heading?"`
+					goto heading
+			`	"Date back thousands of years, the title does. First used by early Saryds, when set out to explore beyond the home system, they had, adopted by the Heliarchs, it was, and remained in use to this day, the term has," she says in excitement, pausing with a smile as if considering that she herself is one such 'inspector' now.`
+			label heading
+			`	"Ready to go outside, we are not. Not for the purpose of finding out about the last expedition, at least. A tool from House Idriss, we need, if to trace my ancestor's steps, we are, so to <destination>, we must go."`
+			`	As she's about to continue, your conversation is cut short by a Kimek rushing towards you two, shouting frantically. Once he's closer, you recognize him as arbiter Kicharri from earlier, and Sedlitaris tries to calm him down while he's catching his breath. "Professor, if about my request this is, already met with the council, I have. Approved it--"`
+			`	"I know! Exactly what I'm furious about, that is!" He shouts. "Asked them to give you a definitive answer, I had, but not this kind of definitive! Listen, permission or not, safe to leave, it is not. Know that, you do." He looks at her for a few moments, then to you. "Captain, as her superior, greatly appreciate your intent to help Sedlitaris, I do, but too brash about this, she has been. Take her out of our space, you must not."`
+			`	He looks at her again, as if expecting some response. When no response comes, he turns away slowly, saying "Don't waste your life on this," and walking away. Sedlitaris sighs once he turns a corridor, and says she's ready for the trip.`
+			choice
+				`	"Hold on, what did he mean by it not being safe to leave? I thought you had permission."`
+				`	"Alright, let's head back to the ship."`
+					goto end
+			`	"I do. Know what he meant, I also do not," she says, looking back to the corner Kicharri turned. "Approving of my wish for this trip, he never was. Some time to calm down, he needs. For now, back to your ship, we should go."`
+			label end
+			`	You find five Heliarchs waiting by your ship, who Sedlitaris introduces as representatives of the council, and who must be present for when House Idriss provides the piece of equipment she wants. You show her and the others to their bunks and prepare for the trip.`
+				accept
+	on visit
+		dialog `You land on <planet>, but realize that your escort carrying Sedlitaris and the Heliarch council representatives hasn't entered the system yet. Better depart and wait for it.`
+
+mission "Coalition: Expeditions Past 3"
```

Comment:
I see the pylon as effectively the same as the Quarg device that they permit Elias to use in helping you locate Giftbringer.
As in, it's a tech piece that's not granted to the player, but to someone trusted by the faction of the person the player will be transporting. Similarly, I imagine both Elias and Sedlitaris keep the devices in their bunks when not in use.
The Quarg tracking thing doesn't have an outfit for it, so I don't think one is needed for the pylon here.

Reply:
Do you think it's big enough to warrant cargo space?

Reply:
Probably would take up space that could otherwise be filled with a cargo crate or two, but it's not something kept in your cargo bay, so I don't see the need for it.